### ISSUE TITLE
loss is very low

### ISSUE THREAD

**Issue by wushilian** [pierluigiferrari/ssd_keras#13]
I use pasocal voc2007 to train the model,and the loss is always is low,like0.2,Is anything wrong?

**Comment by pierluigiferrari** [comment]
Are you trying to train from scratch or with pre-trained VGG-16 weights loaded? In case you tried the former, that's not going to work. Check out the download link for the pre-trained VGG-16 weights in the README. Training SSD300 on Pascal VOC with pre-trained VGG-16 weights works very well, but it takes a lot of training steps until you see half-decent results. Also check out the updated `train_ssd300` notebook: I changed the data augmentation operations, which is important to train successfully.
